sdk/js/src/main/scala/org/ergoplatform/sdk/js/Prover.scala

```diff
@@ -0,0 +1,57 @@
+package org.ergoplatform.sdk.js
+
+import org.ergoplatform.sdk
+import typings.fleetSdkCommon.boxesMod.Box
+import typings.fleetSdkCommon.{commonMod, inputsMod, tokenMod, transactionsMod}
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSExportTopLevel
+
+/** Equivalent of [[sdk.Prover]] available from JS. */
+@JSExportTopLevel("Prover")
+class Prover(_prover: sdk.Prover) extends js.Object {
```

Comment:
too generic name, better to have more descriptive

Reply:
It is generic prover, nothing special about it, so the name is generic. Please suggest.

Reply:
done